docs/master/api-reference/directives.md

```diff
@@ -2892,32 +2892,49 @@ enum Operator {
 
 ### Usage
 
+The most basic way to use this directive is with the dynamic `WhereConstraints` input and the `@whereConstraints` directive with no arguments.
```

Comment:
I intentionally put this at the very end, as i don't really recommend this approach. The default should be to specify the columns.